### ISSUE TITLE
Ultimate ipac led json

### ISSUE THREAD

**Issue by mattjonesc** [katie-snow/Ultimarc-linux#93]
Hi, 
more of a question than an issue:
Are there any other parameters that I can pass to the ipacultimate_led.json to control the Leds using umtool? I was able to get this working using the sample script but would like to configure all buttons and specify the rgb color values.  

Thanks!

**Comment by katie-snow** [comment]
Do you want to configure actions and leds?  The ipacultimate_pins.json has the format for configuring both at the same time.  

You will have to expand the "intensity" array with entries for each pin number.

**Comment by mattjonesc** [comment]
Thank you katie. I’m just trying to configure the colors for each led… for example, button1 connected to led row 1-3 on the board I want to make blue, is there a parameter that I can configure to make R,G,B values = 0,0,16?

**Comment by mattjonesc** [comment]
Hi Katie,

Apologies for my ignorance.  I was able to figure this out.  The led parameter corresponds to the led pin and the intensity is the individual RGB value for each pin.  One thing that I got stuck on was that the led parameter is zero-indexed while the physical board itself (what is printed on it) is not.  Therefore, to set my button connected to the first led row on the board (labeled 1 - 3), I need to specify the led intensities for 0, 1, 2.

As a follow on question, can you explain what the "LED states random" and "fade rate" parameters control and what values can be passed to each?

Thank you again!

**Comment by jimmejames** [comment]
> led parameter is zero-indexed while the physical board itself (what is printed on it) is not. 

This comment was immensely helpful!  I had an LED on pin 5 and had "5" plugged in for the LED... now still have the LED on pin 5, but have "4" plugged into the json file and now control the brightness.


Did you figure out what the "LED states random" and "fade rate" are?

**Comment by mattjonesc** [comment]
> > led parameter is zero-indexed while the physical board itself (what is printed on it) is not.
> 
> This comment was immensely helpful! I had an LED on pin 5 and had "5" plugged in for the LED... now still have the LED on pin 5, but have "4" plugged into the json file and now control the brightness.
> 
> Did you figure out what the "LED states random" and "fade rate" are?

Nope, I just have the "LED states random" set to false and individually set the LED intensities with values between 0-255.  Somewhat crude method, but only way I was able to get it to work for me.
